.gitignore

```diff
@@ -7,6 +7,7 @@ vsql-server
 grammar.bnf
```

Comment:
Actually, I forgot to remove these as well:

```
grammar.bnf
vsql/grammar.v
```

Also, `scripts/generate-grammar` does not need to be added?

Reply:
`scripts/generate-grammar` should be ignored, this file is automatically built on .vsh run, no reason to keep it in git so binary is dynamically-linked and not portable.